spec/audited/audit_spec.rb

```diff
@@ -3,6 +3,61 @@
 describe Audited::Audit do
   let(:user) { Models::ActiveRecord::User.new name: "Testing" }
 
+  describe "audit class" do
+    context "when a custom audit class is configured" do
+      it "should be used in place of #{described_class}" do
+        class CustomAudit < Audited::Audit
+          def custom_method
+            "I'm custom!"
+          end
+        end
+
+        Audited.config { |config| config.audit_class = CustomAudit }
+
+        class TempModel < ::ActiveRecord::Base
+          self.table_name = :companies
+          audited
+        end
+
+        record = TempModel.create
+
+        audit = record.audits.first
+        expect(audit).to be_a CustomAudit
+        expect(audit.custom_method).to eq "I'm custom!"
+
+        Audited.config { |config| config.audit_class = nil }
```

Comment:
These cleanup sections should be in an `after` block, rather than the main spec body. (If possible, also move the new classes and setup to a `before` too.)